webapp/app.py

```diff
@@ -523,15 +523,36 @@ def utility_processor():
 )
 
 app.add_url_rule("/getubuntu/releasenotes", view_func=releasenotes_redirect)
+
+
+from flask_limiter import Limiter
+from flask_limiter.util import get_remote_address
+
+
+limiter = Limiter(
+    key_func=get_remote_address,
+    app=flask.g
+)
+
+class TestView(flask.views.View):
+    decorators = [limiter.limit("2/minute")]
```

Comment:
Need to decide what limit

Reply:
Yeah this is a good initial setting for testing, but how should we decide a good limit?

I spent a little while looking into Graylog for the IPs you asked IS to block:

For 78.128.113.62, we got between 25 and 238 per minute, for 11 minutes between 09:05 and 09:26 on January 31st:

![image](https://user-images.githubusercontent.com/519935/217556982-86dc9944-cdf7-47d4-9e52-a6827ed9cfd3.png)

For 5.196.153.58 it was more sustained over 24 hours between 08:57 on 31st January to 07:27 on 1st February, peaking at 1300/minute, lowest was 3/minute, but probably averaging out around 50/minute:

![image](https://user-images.githubusercontent.com/519935/217557593-ca8b7f0f-283e-4131-8ebc-5e72cbfc48e3.png)

For 103.212.98.234, it was more than 24 hours between 08:10 on 5th February to 10:40 on 6th February. The peak was 949/minute, the low was about 2/minute, and it had this spiking quality, with a spike of around 200/minute every 10 minutes or so.

![image](https://user-images.githubusercontent.com/519935/217559151-a331e670-68fc-4b7e-98ee-0b52e8d7f2bb.png)

I wonder if it's possible to set multiple thresholds? Maybe we should have a threshold of around 50/minute, but maybe maximum per 10 minutes and per hour?